accepted/0011-downstream-keyer.md

```diff
@@ -0,0 +1,39 @@
+- Start Date: 2020-01-23
+- RFC PR: #11
+- Mantis Issue: https://obsproject.com/mantis/view.php?id=602
+
+- Related PRs:
+	1. https://github.com/obsproject/obs-studio/pull/1321
+
+# Summary
+
+Add the ability for users to create global scenes thats are on top of every scene.
+
+# Motivation
+
+Users won't have to create meta scenes and copy them to every scene.
+
+# Detailed design
+
+## User UX
+
+- Have an extra dock for the DSK scene (same as the scenes dock)
+- Have the dock on by default, so users easily know about it
+- Users could switch between DSK channels with combo box in the dock
+- Alternatively, use hotkeys to change DSK channels, so we don't clog up the UI
+- There would be an extra option in the combo box for disabling the DSK
+- Call the DSK "Global Scenes" so users would more likely know what it is
```

Comment:
Because it's more of an advanced feature, I don't think we necessarily need to dumb down the name.

Reply:
I agree it's a more advanced feature but when users realise this is the way to do channel bugs etc. I suspect it will get more use, and thats more likely if it's called global rather than DSK.

It might be interesting in future to add a notification when a user adds a source to multiple scenes (3+) the first time to show a dialog that tells them about global scenes and offers to enable it for them.